app/views/foreclosures/show.html.erb

```diff
@@ -1,6 +1,78 @@
 
 <h2>Foreclosure</h2>
 <p>See any information that's incorrect below? Simply click the data and change it out.</p>
-<%= render "foreclosure" %>
-
-<%= link_to "Delete this application.", "/foreclosures/#{@foreclosure.id}", method: :delete, data:{confirm: "Are you sure you want to delete this foreclosure application?"} %>
+<div class="container-fluid">
+  <% flash.each do |key, value| %>
+    <div class="alert alert-<%= key %> alert-dismissible face show" role="alert">
+      <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+      <%= value %>
+    </div>
+  <% end %>
+  <table class="table table-hover">
```

Comment:
can you do something about this insane amount of indentation? Blast it in the face!!!